functions/src/replication/utils.ts

```diff
@@ -0,0 +1,24 @@
+import { createClient, PostgrestResponse } from '@supabase/supabase-js'
+
+import { DEV_CONFIG } from '../../../common/envs/dev'
+import { PROD_CONFIG } from '../../../common/envs/prod'
+import { isProd } from '../utils'
+
+export function createSupabaseClient() {
```

Comment:
Shouldn't this file be under the `supabase` directory?

Reply:
I think as long as we have a `src` directory, ource code should be in it and non-source code shouldn't. If this was in the `supabase` directory, we would have to change the build script to build it and package it.